libs/platform/src/lib/components/table/table.component.scss

```diff
@@ -1,42 +1,206 @@
+        }
+    }
+
+    .fdp-table__body-table,
+    .fdp-table__header-table {
+        // table-layout fixed does not work with absolutely positioned cells
+        // instead we need to keep each cell with defined width
+        table-layout: auto;
+        width: 100%;
+        // need to keep borders separate in order
+        // to align cells which are absolutely positioned
+        border-collapse: separate;
+        border-spacing: 0;
+    }
+
+    .fd-table__body,
+    .fd-table__header,
+    .fd-table__row {
+        border: none !important;
+    }
+
     .fd-table__cell {
-        max-width: 200px;
+        box-sizing: border-box;
+
         white-space: nowrap;
         overflow: hidden;
         text-overflow: ellipsis;
-    }
 
-    .fd-table__cell--fixed {
-        min-width: 200px;
         max-width: 200px;
+        min-width: 200px;
+    }
+
+    .fdp-table__cell--selection {
+        min-width: 2.8rem;
+        max-width: 2.8rem;
+
+        padding: 0 !important;
+    }
+
+    .fd-table__row,
+    .fd-table__cell {
+        height: 2.75rem;
+    }
+
+    &.fd-table--compact,
+    &.fd-table--condensed {
+        .fd-table__row,
+        .fd-table__cell {
+            height: 2rem;
+        }
+
+        .fdp-table__cell--selection {
+            min-width: 2rem;
+            max-width: 2rem;
+        }
+    }
+
+    .fdp-table__container {
+        position: relative;
     }
 
-    .fd-table__cell--has-menu {
-        cursor: pointer;
+    @mixin fdp-empty-scrollbar-block() {
+        box-sizing: border-box;
+        background-color: #f2f2f2;
+        background-color: var(--sapList_HeaderBackground, #f2f2f2);
     }
 
-    .fd-table__header {
+    .fdp-table__header {
+        position: relative;
+
+        overflow: hidden;
+
+        border: 0.0625rem solid #e4e4e4;
+        border: var(--sapList_BorderWidth, 0.0625rem) solid var(--sapList_BorderColor, #e4e4e4);
+        border-bottom: none;
+
+        .fdp-table__scrollbar-mock {
+            position: absolute;
+            top: 0;
+            right: 0;
+            height: 100%;
+
+            @include fdp-empty-scrollbar-block();
+
+            @include fd-rtl() {
+                right: auto;
+                left: 0;
+            }
+        }
+
+        @include fd-rtl() {
+            padding-right: 0;
+
+            .fdp-table__scrollbar-mock {
+                right: auto;
+                left: 0;
+            }
+        }
+
+        .fd-table--fixed {
+            overflow: hidden;
+        }
+
+        .fd-table__cell {
+            border-bottom: none;
+        }
+
         .fd-table__popover--custom {
             display: none;
         }
+
+        .fdp-table__cell--has-menu {
+            cursor: pointer;
+        }
     }
 
-    .fd-table__row--group {
+    .fdp-table__body {
+        position: relative;
+
+        overflow: hidden;
+        overflow-y: scroll;
+
+        .fd-table--fixed {
+            overflow: hidden;
+            overflow-x: scroll;
```

Comment:
Is there any reason to not having `overflow-x/y  auto`?

Reply:
Yeah,
The trick is the parent has only vertical scroll and the child has only horizontal scroll

Reply:
I probaly got you wrong.
We have mock scrollbar area in the header and need to track the scrolbar width and set it there in order to have the same header width as the body width. 
So just for simplicity I added `overflow-x: scroll` in order to keep this width as a constant.
In the future we could improve it and add a proper tracking of it.